modules/tensor_mechanics/src/materials/ADRadialReturnStressUpdate.C

```diff
@@ -117,7 +115,7 @@ Real
 ADRadialReturnStressUpdate<compute_stage>::computeReferenceResidual(
     const ADReal & effective_trial_stress, const ADReal & scalar_effective_inelastic_strain)
 {
-  return MetaPhysicL::raw_value(effective_trial_stress) / _three_shear_modulus -
+  return MetaPhysicL::raw_value(effective_trial_stress / _three_shear_modulus) -
```

Comment:
The parenthesis was set on purpose to avoid the division of a dual number by a real as `_three_shear_modulus` is always just a Real.

Reply:
Oh, you changed that. Sorry.

Reply:
`_three_shear_modulus` isn't necessarily always constant, since it can depend on temperature, right?